test/functional-test/data/output/2013/01/network/degree_node_number_csv/in_degree

```diff
@@ -0,0 +1,263 @@
+(lp0
```

Comment:
do we need these data files? Can't we reuse the existing data files in tests/data?